copyvios/checker.py

```diff
@@ -144,6 +144,10 @@ def _perform_check(query, page, use_engine, use_links):
             _LOGGER.exception("Failed to retrieve cached results")
 
     if not query.result:
+        if use_engine and not query.requester_username:
+            query.error = "not logged in"
```

Comment:
Instead of making this an error, can we redirect them to `/login?next=...`?

Reply:
Done. I've also added some extra interface text at `/login` to explain what the login is for and where they'll be sent to next (whether it be `/?action=search&...` or some other place). Message from `index.html` removed since it won't be needed anymore.